vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/AbstractElementAction.java

```diff
@@ -139,4 +140,14 @@ public void setRetrySearchIfStale(boolean retrySearchIfStale)
     {
         this.retrySearchIfStale = retrySearchIfStale;
     }
+
+    protected boolean isNormalizeSpace()
+    {
+        return normalizeSpace;
```

Comment:
couldn't it be a part of the search parameters along with visibility?

Reply:
> couldn't it be a part of the search parameters along with visibility?

I think it's possible
But SearchParameters isn't a bean, this will entail even more changes